test_generators/operations/main.py

```diff
@@ -1,9 +1,31 @@
+from eth2spec.test.block_processing import (
+    test_process_attestation,
+    test_process_attester_slashing,
+    test_process_block_header,
+    test_process_deposit,
+    test_process_proposer_slashing,
+    test_process_transfer,
+    test_process_voluntary_exit
+)
+
 from gen_base import gen_runner
 
-from deposits import mini_deposits_suite, full_deposits_suite
+from suite_creator import generate_from_tests, create_suite
 
 if __name__ == "__main__":
     gen_runner.run_generator("operations", [
```

Comment:
The pattern could drop the explicit create_suite per line and instead pass the creator in as a separate argument to run_generator.

In your assessment is the create_suite pattern fixed enough to do this to drop the verbosity a bit here? or do you think the gen_runner should not be concerned about how suites are created?

Reply:
I prefer a little bit of verbosity here, as it gives us a lot of flexibility: we can just add tests to any particular type by concatenating more to the list returned by `generate_from_tests`. Or if necessary, modify the returned list, to e.g. exclude a given test.